bokehjs/src/coffee/models/canvas/cartesian_frame.coffee

```diff
@@ -42,24 +42,21 @@ export class CartesianFrame extends LayoutCanvas
         ranges[name] = extra_range
     return ranges
 
-  _get_scales: (scale_type_name, ranges, frame_range) ->
+  _get_scales: (scale, ranges, frame_range) ->
```

Comment:
There's some weirdness here:

By building this 'scales' dict, we're implicitly supporting having a LinearScale and CategoricalScale (or LogScale and CategoricalScale), but not a LinearScale and LogScale. It's not clear (to me) why we'd make that distinction.